openapi/cosmos_v2.yaml

```diff
@@ -874,29 +874,64 @@ components:
 
 # Biomedical Concept Categories
     bcBiomedicalConceptsCategories:
+      type: object
+      properties:
+        name:
+          type: string
+          example: Biomedical Concept Categories
+        label:
+          type: string
+          example: Biomedical Concept Categories List
+        _links:
+          $ref: '#/components/schemas/bcBiomedicalConceptsCategoriesLinks'
+
+    bcBiomedicalConceptsCategoriesLinks:
       type: object
       properties:
         categories:
           type: array
           items:
-            type: string
-          example:
-            - Vital Signs
-            - Laboratory Tests
-            - Chemistry Tests
-            - Gas Measurements
+            $ref: '#/components/schemas/bcBiomedicalConceptsCategoryRef'
+        self:
+          $ref: '#/components/schemas/bcBiomedicalConceptsCategoriesRef'
+
+    bcBiomedicalConceptsCategoryRef:
```

Comment:
Should this include the actual category name as the component? Maybe as the title? The reason I ask is if we want to support a UI with this endpoint it would be nice to be able to easily get the category name from these links. It could be an additional property if necessary.